extract-variables/apiproxy/policies/AM-Set-Headers.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage continueOnError="false" enabled="true" name="AM-Set-Headers">
+  <DisplayName>AM-Set-Headers</DisplayName>
+  <Set>
+    <Headers>
+      <!--Apigee flow variables-->
+      <Header name="system.timestamp">{system.timestamp}</Header>
+      <Header name="system.time">{system.time}</Header>
+      <Header name="organization.name">{organization.name}</Header>
+      <Header name="apiproxy.name">{apiproxy.name}</Header>
+      <Header name="apiproxy.revision">{apiproxy.revision}</Header>
+      <Header name="proxy.basepath">{proxy.basepath}</Header>
+      <Header name="proxy.name">{apiproxy.name}</Header>
+      <Header name="proxy.pathsuffix">{proxy.pathsuffix}</Header>
+      <Header name="message.headers.count">{message.headers.count}</Header>
+      <Header name="message.headers.names">{message.headers.names}</Header>
+      <Header name="client.ip">{client.ip}</Header>
+      <Header name="request.uri">{request.uri}</Header>
+      <Header name="request.headers.names">{request.headers.names}</Header>
+      <Header name="request.header.user-agent.values">{request.header.user-agent.values}</Header>
+      <Header name="request.path">{request.path}</Header>
+      <Header name="request.querystring">{request.querystring}</Header>
+      <Header name="request.queryparams.names">{request.queryparams.names}</Header>
+      <Header name="request.verb">{request.verb}</Header>
+      <Header name="target.url">{target.url}</Header>
+      <Header name="target.host">{target.host}</Header>
+      <Header name="target.ip">{target.ip}</Header>
```

Comment:
I don't think we need to set all these variables as headers. I'd just set the ones below which were extracted in the previous steps.